bgpd/bgp_community.c

```diff
@@ -910,3 +910,132 @@ void community_finish(void)
 	hash_free(comhash);
 	comhash = NULL;
 }
+
+static unsigned int bgp_aggr_community_hash_key(void *data)
```

Comment:
community_hash_make?

Reply:
Yes, @donaldsharp.
I just added the new functions just as a safeguard.
In case the older function bgp_aggr_community_hash_cmp() goes through any changes in the future, I didn't want that to have an effect on the aggregation.

Reply:
I added a new function to safeguard the aggregation hash just in case the existing function goes through changes in the future.
I've made the changes to use the existing function.